app/models/questionnaire.rb

```diff
@@ -11,6 +11,11 @@ class Questionnaire < ApplicationRecord
   accepts_nested_attributes_for :questions, allow_destroy: true
 
   scope :active, -> { where(active: true) }
+  scope :by_title_like, ->(title) { where("questionnaires.title ILIKE ?", "%#{title}%") if title.present? }
+  scope :by_kind, ->(kind) { where(kind:) if kind.present? }
```

Comment:
There's no need for the if
kind is already a required argument

the same for the other scopes

Reply:
If I remove the ifs, in the cases where the argument is nil, it will try to find some record that matches the nil argument. For example: when I do Questionnaire.by_kind(nil), it will try to find a questionnaire that has a kind = nil. 

What I want is that in the cases where I pass nil, it will simply not filter anything.

Reply:
I saw how I're calling, got it